usaspending_api/awards/management/commands/load_subawards_sql/150_update_awards.sql

```diff
@@ -0,0 +1,34 @@
+-- Update counts in the awards table with new subaward totals.
```

Comment:
How long does it take for this to run?

I'm assuming it's going to be near constant time every run irregardless of the subaward volume loaded

Reply:
On  `dev`:

- [150_update_awards] finished successfully after 2 m 16 s 650 ms
- [150_update_awards] finished successfully after 2 m 29 s 690 ms
- [150_update_awards] finished successfully after 2 m 43 s 681 ms

Those are with no or very few updates.  If we receive a ton of subawards all at once, I would expect this number to spike a bit.